package/gluon-mesh-batman-adv-core/files/lib/gluon/announce/nodeinfo.d/network/mesh_interfaces

```diff
@@ -1,13 +0,0 @@
-local batman_adv = require 'gluon.batman_adv'
-
-local interfaces = {}
-
-for ifname in batman_adv.interfaces('bat0') do
-  pcall(
-    function()
-      table.insert(interfaces, util.readline(io.open('/sys/class/net/' .. ifname .. '/address')))
-    end
-  )
-end
-
-return interfaces
```

Comment:
`nodeinfo.network.mesh_interfaces` isn't generated by the C version anymore. Obviously, the same information is contained in `nodeinfo.network.mesh.bat0.interfaces`, but in other form. Is this intended? Is it documented? Do we have something like API versions for what is returned by respondd?